examples/ArduinoUnoNano-basic-Australia-AltSoftSerial/ArduinoUnoNano-basic-Australia-AltSoftSerial.ino

```diff
@@ -132,6 +130,8 @@ void initialize_radio()
   }
   Serial.println("Successfully joined TTN. You beauty!");
 
+  // setting the frequency plan to AU915
+  myLora.setFrequencyPlan(TTN_AU);
```

Comment:
I still think this should be done earlier - before the join. We need to know which frequencies/channels to use when doing the join. And during the join the other frequencies might be configured by the network. So the order should be:
1. Configure FP according to what we know.
2. Perform a join.
3. The join accept will define the FP to use.